src/Symfony/Component/PropertyInfo/Extractor/PhpDocExtractor.php

```diff
@@ -128,7 +128,9 @@ public function getTypes($class, $property, array $context = array())
         $types = array();
         /** @var DocBlock\Tags\Var_|DocBlock\Tags\Return_|DocBlock\Tags\Param $tag */
         foreach ($docBlock->getTagsByName($tag) as $tag) {
-            $types = array_merge($types, $this->phpDocTypeHelper->getTypes($tag->getType()));
+            if ($tag && null !== $tag->getType()) {
+                $types = array_merge($types, $this->phpDocTypeHelper->getTypes($tag->getType()));
```

Comment:
The bug doest not exists in 2.7 ?

Reply:
It looks like `PhpDocExtractor` was added in 2.8, and at that time it handled nulls (albeit differently).

https://github.com/symfony/symfony/commit/f1eb1852364d36e51d4f4480c52e8251405d830c#diff-72b9199530f645acef692f3d77d4de4d

Reply:
It may be a good idea to send an extra PR that adds this test case to the `2.8` branch to prevent regressions there.